sklearn/utils/metaestimators.py

```diff
@@ -80,22 +80,22 @@ def if_delegate_has_method(delegate):
     ...         self.better_sub_est = better_sub_est
     ...
     ...     @if_delegate_has_method(delegate='sub_est')
-    ...     def predict(self, X):
-    ...         return self.sub_est.predict(X)
+    ...     def predict(self):
+    ...         pass
     ...
+    >>> class MetaEstTestTuple(MetaEst):
```

Comment:
I would rather this kind of variation and testing to be in unit tests in a different file, not doctests.